packages/api/src/presentation/controllers/users.controller.ts

```diff
@@ -0,0 +1,24 @@
+import { Body, Controller, Post, Res } from '@nestjs/common'
+import { CreateUser } from '@/domain/usecases/db/users/create-user'
+import { Response } from 'express'
+
+@Controller('/user')
+export class UserController {
+  constructor(private readonly createUser: CreateUser) {}
+
+  @Post('/signup')
+  async signup(@Body() body, @Res() response: Response) {
+    const result = await this.createUser.execute(body)
+
+    const responseResult = {
+      statusCode: 204,
+      message: result.message,
```

Comment:
chatice: HTTP Code 204 (No Content) não deve ter corpo. RFC: `A 204 response is terminated by the end of the header section; it cannot contain content or trailers.`